drivers/network/ndis/ndis/miniport.c

```diff
@@ -2399,7 +2399,26 @@ NdisIDispatchPnp(
         Irp->IoStatus.Information |= Adapter->NdisMiniportBlock.PnPFlags;
         break;
 
+      case IRP_MN_REMOVE_DEVICE:
+        Status = IoSetDeviceInterfaceState(&Adapter->NdisMiniportBlock.SymbolicLinkName, FALSE);
```

Comment:
About this `IoSetDeviceInterfaceState` call: is it necessary to redo it here?
Maybe an alternative question would be: Does the PnP sends first an `IRP_MN_STOP_DEVICE` irp, before sending the `IRP_MN_REMOVE_DEVICE` irp?
I'm asking this, because the implementation of `IRP_MN_STOP_DEVICE` in `NdisIPnPStopDevice` already has such a call:
https://github.com/reactos/reactos/blob/999502cfc47b62956a3431d2137c6391e646c470/drivers/network/ndis/ndis/miniport.c#L2238-L2240